app/graphql/types/query_type.rb

```diff
@@ -8,10 +8,26 @@ class QueryType < Types::BaseObject
     # They will be entry points for queries on your schema.
 
     # TODO: remove me
-    field :test_field, String, null: false,
-      description: "An example field added by the generator"
-    def test_field
-      "Hello World!"
+    # field :test_field, String, null: false,
+    #   description: "An example field added by the generator"
+    # def test_field
+    #   "Hello World!"
+    # end
+
+    field :users, [Types::UserType], null: false,
+      description: "return all users"
+
+    field :user, Types::UserType, null: false do
+      description "return a user"
+      argument :id, ID, required: true
+    end
+
+    def users
```

Comment:
not sure when we would need this...

Reply:
no had it in there to get the hang of Graphql can delete on next PR